addons/interactions/src/preset/argsEnhancers.ts

```diff
@@ -1,11 +1,11 @@
 import { Args, addons } from '@storybook/addons';
-import { window as globalWindow } from 'global';
 import { FORCE_REMOUNT, STORY_RENDER_PHASE_CHANGED } from '@storybook/core-events';
 import { AnyFramework, ArgsEnhancer } from '@storybook/csf';
 import { instrument } from '@storybook/instrumenter';
 import { ModuleMocker } from 'jest-mock';
 
-const { fn } = new ModuleMocker(globalWindow);
+const JestMock = new ModuleMocker(global);
```

Comment:
Ugh, I screwed this up, it needed to be `globalThis`.  I'll need to send another PR to fix this.